### PR TITLE
modified the .eslintrc file and and eslint rules to ES6 2020

### PR DESCRIPTION
**Pull request by raj5036**
Updated the codebase as pre issue #2 .Please check it .

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "parserOptions": {
+    "ecmaVersion": 11,
+    "sourceType": "module",
+    "ecmaFeatures": {
+        "jsx": true
```
Comment (elit-altum) [inline .eslintrc:6]: Any reasons for this? JSX is preferred for react only I believe. We want to keep this module as flexible and un-opinionated as possible.

**Review by elit-altum — CHANGES_REQUESTED** [review]

**Comment by elit-altum** [comment]
@raj5036 Do you want this PR to specifically count for Hacktoberfest? If that's the case, I'd merge it on 1st October only.

**Comment by raj5036** [comment]
OK @elit-altum  I am making the change. I added the jsx option mistakenly , but ok I will remove it.

And yes I want this PR to count for Hacktoberfest. So please merge it on 1st October .

**Comment by Max19thl** [comment]
I am making the change. I added the jsx option mistakenly , but ok I will remove it. yes I want this PR to count for Hacktoberfest. So please merge it on 1st October .

**Comment by elit-altum** [comment]
@raj5036 Please resolve conflicts

**Comment by raj5036** [comment]
Is it Okay now ?

**Comment by raj5036** [comment]
@elit-altum Is it Okay now? Or should I delete this PR and a  create a new one?

**Pull request merged**